dev/dune

```diff
@@ -9,6 +9,7 @@
 
 (rule
   (targets dune-dbg)
+  (package coq)
```

Comment:
This seems unrelated, why do you want to install these objects ?

Reply:
I don't want to install them. But without these additional `(package coq)`, I got errors when running `dune build -p coq-doc` like:

```
File "dev/shim/dune", line 28, characters 4-20:
28 |   %{bin:coqtop.byte}
         ^^^^^^^^^^^^^^^^
Error: Program coqtop.byte not found in the tree or in PATH
 (context: default)
```

Reply:
Still, I reverted these changes.

Reply:
Oh, this is due to `-p` assuming the dependencies to be already built; even so that error seems strange, let me see.

Reply:
FTR it also happens when trying to install through opam:

```
### output ###
# File "dev/dune", line 39, characters 6-44:
# 39 |     %{lib:coq.top_printers:top_printers.cmi}
#            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Library "coq.top_printers" not found.
# Hint: try: dune external-lib-deps --missing -p coq-doc -j 3 @install
```